packages/backend/src/ee/services/ai/tools/findExplores.ts

```diff
@@ -15,104 +10,111 @@ type Dependencies = {
 
 const PAGE_SIZE = 15;
 
-const generateExploreResponse = (
-    table: CatalogTable,
-    fields: CatalogField[],
-) => {
-    const allTableNames = [table.name, ...(table.joinedTables ?? [])];
-
-    const dimensions = fields
-        .filter((field) => field.fieldType === FieldType.DIMENSION)
-        .filter((field) => allTableNames.includes(field.tableName))
-        .sort((a, b) => (b.chartUsage ?? 0) - (a.chartUsage ?? 0));
-
-    const metrics = fields
-        .filter((field) => field.fieldType === FieldType.METRIC)
-        .filter((field) => allTableNames.includes(field.tableName))
-        .sort((a, b) => (b.chartUsage ?? 0) - (a.chartUsage ?? 0));
-
-    return `
-<Explore table="${table.name}">
+const generateExploreResponse = ({
+    table,
+    dimensions,
+    metrics,
+    dimensionsPagination,
+    metricsPagination,
+    shouldTruncate,
+}: Awaited<ReturnType<FindExploresFn>>['tablesWithFields'][number] & {
+    shouldTruncate: boolean;
+}) =>
+    `
+<Explore tableName="${table.name}">
     <Label>${table.label}</Label>
     <BaseTable alt="ID of the base table">${table.name}</BaseTable>
 
     ${
         table.aiHints && table.aiHints.length > 0
             ? `
-    <AI Hints>
-        ${table.aiHints.map((hint) => `<Hint>${hint}</Hint>`).join('\n')}
-    </AI Hints>`.trim()
+    <AIHints>
+        ${table.aiHints
+            .map((hint) => `<Hint>${hint}</Hint>`)
+            .join('\n        ')}
+    </AIHints>`.trim()
```

Comment:
thought: I'm wondering if `AIHints` is "confusing" and `<Description priority="high"/"low" />` would work better. Might be worth it to do an eval for this